File: dotnet-api/Models/UserModel.cs
namespace DotnetApi.Models
{
    public class UserModel : ModelBase
    {
        public string UserName { get; set; }

        public string FirstName { get; set; }

        public string LastName { get; set; }

        public string Fullname
        {
            get { return $"{FirstName} {LastName}".Trim(); }
        }

        public string Street { get; set; }

        public string City { get; set; }

        public string Zip { get; set; }

        public string Country { get; set; }

        public string Email { get; set; }

        public string PhoneNumber { get; set; }
    }
}

File: go-api/internal/models/user_model.go
package models

type UserModel struct {
	BaseModel
	UserName    string `json:"userName"`
	FirstName   string `json:"firstName"`
	LastName    string `json:"lastName"`
	Street      string `json:"street"`
	City        string `json:"city"`
	Zip         string `json:"zip"`
	Country     string `json:"country"`
	PhoneNumber string `json:"phoneNumber"`
	Email       string `json:"email"`
} //@name User

type SaveUserModel struct {
	UserName    string `json:"userName" validate:"required,uniqueUserName"`
	FirstName   string `json:"firstName" validate:"required"`
	LastName    string `json:"lastName"`
	Street      string `json:"street"`
	City        string `json:"city"`
	Zip         string `json:"zip"`
	Country     string `json:"country"`
	PhoneNumber string `json:"phoneNumber"`
	Email       string `json:"email"`
} //@name SaveUser
